base/pkg/entry.jl

```diff
@@ -212,6 +215,8 @@ function clone(url_or_pkg::AbstractString)
         pkg = url_or_pkg
         url = readchomp(urlpath)
         # TODO: Cache.prefetch(pkg,url)
+    elseif !(':' in url_or_pkg) && !isfile(urlpath)
+        throw(PkgError("package $url_or_pkg doesn't appear to exist"))
```

Comment:
Are these (especially the one above) effectively disables cloning a repo from a local path without the `file://` prefix?

This can still trigger the issue if you are cloning from `file://$(Pkg.dir("WTF"))` although I'm not sure how important that is.

This doesn't work around the bug in `LibGit2` either (i.e. you will see this issue if you call `LibGit2.clone("WTF", "WTF")`). Do we expose the `LibGit2` module as an API and should we workaround upstream bugs in it?